Formula/cypher-shell.rb

```diff
@@ -0,0 +1,24 @@
+class CypherShell < Formula
+  desc "Command-line shell where you can execute Cypher against Neo4j"
+  homepage "https://github.com/neo4j/cypher-shell"
+  url "https://github.com/neo4j/cypher-shell/releases/download/1.1.9/cypher-shell.zip"
+  version "1.1.9"
+  sha256 "027dd83ff10ac24077dc9ab7e1be4a624f8cadeab4039bb6800584d94fecf49f"
+
+  depends_on :java => "1.8"
+
+  def install
+    rm_f Dir["bin/*.bat"]
+
+    ENV["NEO4J_HOME"] = share
+    share.install ["cypher-shell.jar"]
+
+    bin.install ["cypher-shell"]
+    bin.env_script_all_files(share, :NEO4J_HOME => ENV["NEO4J_HOME"])
+  end
+
+  test do
+    ENV["NEO4J_HOME"] = libexec
+    system "#{bin}/cypher-shell", "--version"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (`foo --version` or `foo --help`) are not sufficient, as explained in the [formula cookbook](https://docs.brew.sh/Formula-Cookbook).

In most cases, a good test would involve running a simple test case: run `#{bin}/foo input.txt`.
- Then you can check that the output is as expected (with `assert_equal` or `assert_match` on the output of `shell_output`)
- You can also check that an output file was created, if that is expected: `assert_predicate testpath/"output.txt", :exist?`

Some advice for specific cases:
- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.